index.html

```diff
@@ -33,40 +42,63 @@ <h2 class="sub-t">I’am a Software Developer</h2>
                 </ul>
             </div> 
         </section> 
-        <section>
-            <h2 class="myrr">My Recent Work</h2>
-            <img src="./images/Vecto1.png" alt="blackline" class="blackline">
+        <section class="description-med">
+            <div >
+                <h2 class="myrr">My Recent Work</h2>
+                <img src="./images/Vecto1.png" alt="blackline" class="blackline">
+            </div>
             <div class="img-pp"> <img src="./images/Img Placeholder.png"></div>
-            <div class="description">
-                <h2 class="multi">Multi-Post Stories</h2>
-                <p class="daily">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-                <ul class="languages">
-                    <li class="g-item">html</li>
-                    <li class="g-item">css</li>
-                    <li class="g-item">bootstrap</li>
-                    <li class="g-item">ruby</li>
-                </ul>
-                <button type="button" class="but1" >See project</button>
+                <div class="description">
+                    <h2 class="multi">Multi-Post Stories</h2>
+                    <p class="daily">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <ul class="languages">
+                        <li class="g-item">html</li>
+                        <li class="g-item">css</li>
+                        <li class="g-item">bootstrap</li>
+                        <li class="g-item">ruby</li>
+                    </ul>
+                    <button type="button" class="but1" >See project</button>
```

Comment:
- [ ] [Optional] You should stick to the design as much as possible. please fix this spacing issue between the first cards and the 3 cards. See the image below for reference
### Your design
![your](https://user-images.githubusercontent.com/81727988/213692398-aea98b9a-eb24-416d-8740-a0f07e6eabfb.jpg)


### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/213692432-ba58ce4e-9a0d-498c-bca3-d2ec47c45ad3.jpg)